src/components/AddressSearch/index.js

```diff
@@ -208,14 +208,15 @@ function AddressSearch(props) {
 
             // Autocomplete returns any additional valid address fragments (e.g. Apt #) as subpremise.
             street2: subpremise,
-
+            // Make sure country is updated first, since city and state will be reset if the country changes
+            country: '',
```

Comment:
@Ollyws This minor change is to make sure that when selecting address, the country is updated first. Because state and city are reset after country change

Reply:
@Ollyws > I think this breaks the country selection when you select an address, notice the country remains the same
I think this bug is related to PR https://github.com/Expensify/App/pull/26742. Just tested in staging https://staging.new.expensify.com/ and encountered the same issue

Reply:
Sure. Will report when i have a chance